### PR TITLE
Updating an Existing Tool

### PR DESCRIPTION
**Pull request by kpbioteam**

### CODE REVIEW

File: epigenetics.yaml.lock [hunk 1]
```diff
@@ -42,10 +42,13 @@ tools:
 - name: chipeakanno_annopeaks
   owner: kpbioteam
   revisions:
+  - 104882a2de00
   - 6f52c0dc56d4
   tool_panel_section_label: Test Tools
 - name: ewastools
   owner: kpbioteam
   revisions:
+  - 369fef559cfc
```
Comment (bgruening) [inline epigenetics.yaml.lock]: @kpbioteam are both revisions here needed or just the latest one?

Comment (kpbioteam) [reply epigenetics.yaml.lock]: @bgruening just the latest one

**Comment by bgruening** [comment]
removed the older one ...

**Comment by bgruening** [comment]
@kpbioteam when do you need it? Is Saturday ok or sooner?

**Pull request merged**